src/Core/src/Exception/NonParsableResponse.php

```diff
@@ -4,6 +4,6 @@
 
 namespace AsyncAws\Core\Exception;
 
-class ParseResponse extends \RuntimeException implements Exception
```

Comment:
Tha't a BC break isn't it?
What if people use `catch (ParseResponse $e)`
unless the class is internal

Reply:
Yes, it is a BC break. 
But we have not released this class yet. So this is our last chance =)